### PR TITLE
Simple scrollbar

### PR DESCRIPTION
**Pull request by shayanbinary**

### CODE REVIEW

File: sass/components/_categorical-display.scss [hunk 1]
```diff
@@ -112,18 +115,18 @@
...
-        position: absolute;
+        line-height: 40px;
         top: 0;
         @include themify($themes) {
             color: themed('CatDisplayCatTitleText');
         }
 
         &.fixed {
-            position: fixed;
+            position: sticky;
```
Comment (shayanbinary) [inline sass/components/_categorical-display.scss]: for more info https://developer.mozilla.org/en-US/docs/Web/CSS/position

File: sass/components/_view.scss [hunk 2]
```diff
@@ -47,13 +47,15 @@
...
+            position: relative;
 
             .ciq-list {
-                height: 115px;
-                overflow: auto;
-                display: block;
+                max-height: 110px !important;
 
+                .ciq-list-inner {
+                    min-height: 131px;
```
Comment (brucebinary) [inline sass/components/_view.scss]: this line is causing the scrollbar to show up in templates eventhough there's nothing there. Why did you add this?

File: src/store/CategoricalDisplayStore.js [hunk 3]
```diff
@@ -140,7 +153,10 @@ export default class CategoricalDisplayStore {
                 }
             }
         }
-        this.activeHeadTop = this.scrollPanel.offsetTop;
+        if (this.scrollPanel) {
```
[the remark this answers is not in the document]
Comment (shayanbinary) [reply src/store/CategoricalDisplayStore.js]: you are right, `this.scrollPanel` initial before `init` method

File: sass/components/_categorical-display.scss [hunk 4]
```diff
@@ -112,23 +113,36 @@
...
             color: themed('CatDisplayCatTitleText');
         }
 
-        &.fixed {
-            position: fixed;
-            width: 100%;
-            z-index: 1;
-            top: 0px;
+        &.stikcy-top,
+        &.stikcy-bottom {
```
Comment (brucebinary) [inline sass/components/_categorical-display.scss]: typo: sticky not stikcy

File: sass/components/_view.scss [hunk 5]
```diff
@@ -84,15 +103,23 @@
                         border-top: none;
                         margin-top: 0;
                     }
-                    &:nth-last-child(-n+3) {
+                    &:last-child() {
```
Comment (brucebinary) [inline sass/components/_view.scss]: nice touch =)

**Comment by shayanbinary** [comment]
@brucebinary  I just checked again, the only place I changed the default style of `simplebar` is **View** component, and as simplebar make inline style so I have to use `!important` for some cases(Ex, iPhone, and Edge)
